doc/api/tty.md

```diff
@@ -88,6 +88,10 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* Terminal window resize events are unreliable on Windows. The event 
+handler is only executed for changes in the width of the window and the 
```

Comment:
On Windows the console size is the virtual buffer size, not the size of the console window itself. If you shrink window height, the buffer size will remain the same, so no event will be fired. If you enlarge window enough that it will be higher than the buffer, Windows will update the buffer size and event will be signaled.

On Windows the event reliably detected only if the console is read (e.g. `process.stdin.resume();` or `process.stdin.on('data',...)`) in raw mode.

If the console is not read, or is read in line mode, this event can still be signaled. This will happen if a terminal sequence that moves the cursor is used. Calling `process.stdin._refreshSize()` will also trigger the event if the console size has been changed.